### DISCUSSION TITLE
Embed Music Assistant into Dashboard (Lovelace)

### DISCUSSION THREAD

**Discussion by TheLexus** [music-assistant/support#594] [Q&A] [answered]
Hi,

is it possible to embed MA into a Dashboard as iFrame? I tried it and it work with a little problem. The Homeassistant Navigation bar will be embedded too, maybe there is another url to be used?

Thx

**Comment by marcelveldt** [comment, marked as answer]
You can try the stand-alone URL but no guarantees it will work:
http(s)://[ha-base-url]/lib/mass

**Reply by loryanstrant** [reply]
This worked perfectly, thank you!
Would love it if it picked up the dark theme though. :-)

**Reply by liquidbear99** [reply]
It only works on desktop. Unfortunately, it's still not functioning in the iOS app for me even with the new update 2022.7.4

**Comment by TheLexus** [comment]
Works like a charm! Thank you!

**Comment by liquidbear99** [comment]
Unfortunately this does not work on the iOS app. On the iOS app when you open the dashboard with that web element it says "What host to connect to" and takes you to a dialogue each time that is not functional.

**Reply by timmelander** [reply]
Same

**Reply by liquidbear99** [reply]
Thanks for the attempts to fix devs! Unfortunately, it's still not functioning for me even with the new update 2022.7.2

**Comment by fribse** [comment]
I would love to see a way to put it into a dashboard as well, I've designed a very nice and compact UI for our mobiles, without menus, so it's not possible to use the Music Assistant from the sidebar (as it's not showing on the cell phone to save real estate). This gives it a hard no WAF.

**Comment by pressurejets** [comment]
I have a way to put to dashboard without the assistant side panel. works on mobile view BUT 
as im using iframe > ingress its stating 401 error if you don't click the music assistant on home assistant first
there as to be a way around it.

**Comment by legovaer** [comment]
What about using the built-in webserver?

`http://[HA_BASE_URL]:8095/#/settings/editcore/webserver`

I enabled this and now I can open the web app using port **8095**. I'm running mass on ha core.

![Screenshot_20240103_112108_Chrome.jpg](https://github.com/music-assistant/hass-music-assistant/assets/5813212/bb5cb686-4d29-4288-b1be-a01083bee379)

**Comment by OzGav** [comment]
This is v1 info

**Reply by roldengarm** [reply]
[The documentation for when running Docker](https://music-assistant.io/faq/how-to/) references the iframe panel which is no longer supported: https://www.home-assistant.io/integrations/panel_iframe/
Instead, it should reference: https://www.home-assistant.io/dashboards/iframe/
Just tested, seems to work fine, although dedicated MA widgets would be even better :) @OzGav
